docs/support-matrix.md

```diff
@@ -0,0 +1,43 @@
+# Supported Storage Providers
+
+Ark supports sending backup data to a variety of object storage providers, as well as volume snapshots using either a plugin for the volume provider, or [restic][4] for filesystem level backups.
+
+## In-tree providers
+
+Ark supports the follow object and volume storage providers in-tree.
+
+| Provider              | Object Storage       | Block Storage                |
+|-----------------------|----------------------|------------------------------|
+| Amazon Web Services   | S3                   | Elastic Block Store          |
+| Google Compute Engine | Google Cloud Storage | Google Compute block devices |
+| Azure                 | Azure Blob Storage   | Azure Compute block devices  |
+
+## Plugins
+
+Plugins provide Ark a mechanism to use object and volume storage from providers not already supported in tree.
```

Comment:
I'd rather say something like "Ark's plugin architecture enables anyone to add support for additional object and block storage platforms without modifying the core Ark codebase.  The following third-party plugins are available for use:"